tests/benchmarks/test_multiplexer.py

```diff
@@ -9,10 +10,17 @@
 IP_ADDR = ipaddress.ip_address("8.8.8.8")
 
 
-def test_process_1000_2048_byte_channel_messages(
+@pytest.mark.parametrize(
+    ("size", "message_count"),
+    [(2048, 1000), (1024 * 1024, 100)],
```

Comment:
It used to be 1000@1MiB.

Reply:
It turned out 100 was enough to eliminate the jitter and it reduces the CI run time. I didn't want to change it before because it meant we would lose the history, but since we are already going to lose the history here it seemed like the right time to do it.